packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -196,6 +216,10 @@
     onChange({ detail: tempFilters }, defKey)
     drawer.hide()
   }
+
+  onMount(async () => {
+    await environment.loadVariables()
```

Comment:
Like other comment, the other calls to `environment.loadVariables` have an error try catch, might be worth adding here.